src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/instances/x64/reduce.cpp

```diff
@@ -678,6 +737,20 @@ const auto params_MultiAxis_5D_Hybrid_fusing_KeepNoDims = testing::Combine(
         testing::ValuesIn(fusingParamsSet_KeepNoDims),
         testing::ValuesIn(additionalConfigFP32()));
 
+const auto params_MultiAxis_4D_dynamic_with_zero_fusing_KeepNoDims = testing::Combine(
+        testing::Combine(
+                testing::ValuesIn(axesZeroDimFusing),
+                testing::Values(ov::test::utils::OpType::VECTOR),
+                testing::Values(false),
+                testing::ValuesIn(reductionTypes()),
+                testing::ValuesIn(inpOutPrc()),
+                testing::Values(ElementType::undefined),
+                testing::Values(ElementType::undefined),
+                testing::ValuesIn(inputShapes_Dynmic_ZeroDim)),
+        testing::Values(emptyCPUSpec),
+        testing::ValuesIn(fusingParamsSet_KeepNoDims),
```

Comment:
I can see the tests for other fusions have been enabled, great! 
Have you found what was the root cause of the failures for fusions other than Swish?

Reply:
I aligned the test cases with the added functionality in Reduce node of CPU plugin, i.e., the cases of empty input with non-empty output, then all fusion test cases passed. I believe the reduced axes should cover all axes with dim zero.